### PR TITLE
Xonda

### PR DESCRIPTION
**Pull request by CJ-Wright**
@gforsyth are you ok with being a maintaner?

### CODE REVIEW

**Comment by CJ-Wright** [comment]
@gforsyth any suggestions for the tests?

**Comment by gforsyth** [comment]
Hey @CJ-Wright -- thanks for putting this together.  I am fine being a maintainer.  

I should probably go ahead and add some real tests for some of the env listing/checking stuff anyway.  But I probably won't get to that for a while, so maybe the thing to do in the interest of having this built would be to add some ridiculous `assert True`?

**Comment by gforsyth** [comment]
Hey @CJ-Wright -- I went ahead and wrote some super basic tests and just released 0.2.1 which includes those

**Comment by CJ-Wright** [comment]
It seems that it is having problems with conda (maybe because it isn't in the root env?).

**Comment by CJ-Wright** [comment]
@conda-forge/staged-recipes ready for review!

File: recipes/xonda/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{% set name = "xonda" %}
+{% set version = "0.2.1" %}
+{% set sha256 = "09eefdf61bc987d1b1522e49fe116650b93d1536c47881ea7a197398beabaffe" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
```
Comment (ocefpaf) [inline recipes/xonda/meta.yaml]: This can be removed. Can creates the `fn` from the URL.

File: recipes/xonda/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  url: https://github.com/gforsyth/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python setup.py install
```
Comment (ocefpaf) [inline recipes/xonda/meta.yaml]: When using `setuptools` you can do `python setup.py install --single-version-externally-managed --record record.txt`. Or better yet, change `setuptools` to `pip` and use `python -m pip install --no-deps --ignore-installed .`

**Comment by CJ-Wright** [comment]
Thank you very much!

**Comment by ocefpaf** [comment]
@CJ-Wright I am quote "outdated" with `staged-recipes`. If you have pending PRs feel free to ping me.

File: recipes/xonda/meta.yaml (hunk 2 above)
Comment (scopatz) [reply recipes/xonda/meta.yaml]: Pip/setuptools tends to fail for xonsh xontribs because it can't handle implicit namespace packages

**Comment by CJ-Wright** [comment]
@ocefpaf will do!

File: recipes/xonda/meta.yaml (hunk 2 above)
Comment (ocefpaf) [reply recipes/xonda/meta.yaml]: Good to know. However, more most Python packages in `conda-forge` I do recommend moving to `pip`, after all it is the standard :wink:

Comment (scopatz) [reply recipes/xonda/meta.yaml]: Yeah it is just unfortunate that the standard does not adhere to the standard library :wink:

**Pull request merged**